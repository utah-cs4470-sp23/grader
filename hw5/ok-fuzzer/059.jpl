time let {} = .9
print ""
time show ( 81.3 )
print "#]"
assert if false then true else sum [ Pm : h ( ) ] HA4. [ - true ][], ""
read image "" to F
write image k.8 ( [] ) to "("
type u.5KO1 = bool
let {} = [ array [ U : 2 ] ! ( ( .53 ) ) , c421x.55 , true ]
write image o5. ( ) to ""
write image X ( [] ) to "[["
time write image ! {} to "';[\;"
print ":,!@."
