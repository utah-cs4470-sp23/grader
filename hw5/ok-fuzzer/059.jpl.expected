(TimeCmd (LetCmd (TupleLValue) (FloatExpr 0)))
(PrintCmd "")
(TimeCmd (ShowCmd (FloatExpr 81)))
(PrintCmd "#]")
(AssertCmd
 (IfExpr
  (FalseExpr)
  (TrueExpr)
  (SumLoopExpr
   Pm
   (CallExpr h)
   (ArrayIndexExpr (ArrayIndexExpr (VarExpr HA4.) (UnopExpr - (TrueExpr))))))
 "")
(ReadCmd "" (VarArgument F))
(WriteCmd (CallExpr k.8 (ArrayLiteralExpr)) "(")
(TypeCmd u.5KO1 (BoolType))
(LetCmd
 (TupleLValue)
 (ArrayLiteralExpr
  (ArrayLoopExpr U (IntExpr 2) (UnopExpr ! (FloatExpr 0)))
  (VarExpr c421x.55)
  (TrueExpr)))
(WriteCmd (CallExpr o5.) "")
(WriteCmd (CallExpr X (ArrayLiteralExpr)) "[[")
(TimeCmd (WriteCmd (UnopExpr ! (TupleLiteralExpr)) "';[\;"))
(PrintCmd ":,!@.")
